pages/_app.tsx

```diff
@@ -186,4 +187,24 @@ const AppRouting = ({ Component, pageProps }: AppProps) => {
   );
 };
 
+interface AuthProps {
+  apiKey?: string;
+}
+AppRouting.getInitialProps = (appContext: AppContext) => {
+  const props = NextApp.getInitialProps(appContext);
+  const authHeader = appContext.ctx.req?.headers.authorization;
```

Comment:
wait, how is this passed in?

Reply:
It's coming from the server when you pass in the `Authorization` HTTP header. Check out https://github.com/RecordReplay/playwright-tests/pull/104/files#diff-4a2dc5597012da2f007b9fe6f3b92dc7efc11c4be73cd04f34a52b08523c62dfR57-R59 for how we could pass it in for playwright.

Reply:
wow! looks great